src/views/PageSpotRequest.vue

```diff
@@ -286,6 +283,23 @@ export default {
                 }
             });
         },
+
+        spotRequestStatusOptions() {
+            return SpotRequestStatus;
+        },
+
+        onStatusUpdate(spotData, status) {
+            console.log('Spotdata, status', spotData, status);
+            // Get the status id from name
+            if (typeof status === 'string') {
+                const foundStatus = this.spotRequestStatusList.find(
```

Comment:
as there is a method in enum to get the labels, there should be a method in that package to return id based on label.
later this can be used by other enums too.